### PR TITLE
(Hiatus) Fix to reserve another image buffer for compressed image

### PR DESCRIPTION
**Pull request by kogaki**
# What's happening?

When loading .tpl, `TRasterCM32P(m_lx, m_ly)` allocates `m_lx x m_ly x 4`Bytes buffer. While it is the size of `decompressed image`, it is reused for `compressed image`.

### CODE REVIEW

File: toonz/sources/common/trasterimage/tcodec.cpp [hunk 1]
```diff
@@ -745,25 +745,8 @@ bool TRasterCodecLZO::decompress(const UCHAR *inData, TINT32 inDataSize, TRaster
 
 	outRas->lock();
 	memcpy(outRas->getRawData(), decompressedBuffer.data(), decompressedBuffer.size());
-	bool rc = true;
-
 	outRas->unlock();
-
-	/*
```
Comment (kogaki) [inline toonz/sources/common/trasterimage/tcodec.cpp]: `safeMode` had no longer worked.

**Comment by shun-iwasawa** [comment]
Can one of the admins verify this patch?

**Comment by turtletooth** [comment]
Should this be closed?

**Comment by RodneyBaker** [comment]
Initiating a review of old PRs to see which ones are still active and/or being worked.
I tagged this PR as 'roadmap' back in March assuming it is still desired but perhaps that is no longer the case?

**Comment by RodneyBaker** [comment]
Closing as abandoned.

**Pull request closed without merging**